matchms/filtering/require_precursor_mz_above_0.py

```diff
@@ -0,0 +1,29 @@
+from ..typing import SpectrumType
+from ..typing import Union
+
+
+def require_precursor_mz_above_0(spectrum_in: SpectrumType
+                                 ) -> Union[SpectrumType, None]:
+
+    """Returns None if there is no precursor_mz or if <=0
+
+    Parameters
+    ----------
+    spectrum_in:
+        Input spectrum.
+    """
+    if spectrum_in is None:
+        return None
+
+    spectrum = spectrum_in.clone()
+
+    precursor_mz = spectrum.get("precursor_mz", None)
+    if precursor_mz is None:
+        return None
```

Comment:
```suggestion
    if precursor_mz is None:
        pepmass = spectrum.get("pepmass", None)
        assert pepmass is None or not isinstance(pepmass[0], (float, int)), \
            "Found 'pepmass' but no 'precursor_mz', please run add_precursor_mz() first."
        return None
```

Reply:
This is only a safety check in case people are note aware of the `add_precursor_mz` step which should come before running your new filter function.